index.html

```diff
@@ -61,20 +62,16 @@ <h1>Theta Chi Family Tree</h1>
   function search() {
     var query = $('#searchbox').val();
     var success = findBrother(query);
-
     // Indicate if the search succeeded or not.
     if (success) {
-      $('#searchbox').css('color', 'black');
+      $('#searchbox').css('background-color', 'white');
     } else {
-      $('#searchbox').css('color', 'red');
+      $('#searchbox').css('background-color', '#EEC4C6'); // red matching flag
     }
   }
-  document.getElementById("searchbox").onkeypress = function(e) {
+  document.getElementById("searchbox").onkeyup = function(e) {
     if (!e) e = window.event;
-    var keyCode = e.keyCode || e.which;
-    if (keyCode === '13' || keyCode === 13 /* Enter */) {
-      search();
-    }
+    search();
```

Comment:
Can you wrap this in a timeout, along the lines of [this stackoverflow](https://stackoverflow.com/a/7849308)?

---

"Instant search" can feel jarring when typing quickly. Try out "westman", "garcia", or "johnny" to get an idea of what I mean. This timeout should smooth out the user experience, while still being nearly instantaneous.